tmp_script.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+# Rung by passing in the function name to call as first parameter to the script.
+# E.g.  ./tmp_script.sh build_rust
+#
+# Be aware that this script will in a follow up PR be replaced with an ob runner.
+
+set -eux
+cd "$(dirname $0)"
+
+function install_async_bindgen_cli() {
+    cargo install \
+        --git https://github.com/xaynetwork/xayn_async_bindgen.git \
+        async-bindgen-gen-dart \
+        "$@"
+}
+
+function gen_dart_genesis_extension() {
+    cd discovery_engine_core
+    # Doesn't respect cargo config option, but good enough for us for now
+    "${CARGO_INSTALL_ROOT:-${CARGO_HOME:-$HOME/.cargo}}/bin/async-bindgen-gen-dart" \
+        --ffi-class XaynDiscoveryEngineBindingsFfi \
+        --genesis ../discovery_engine/lib/src/ffi/genesis.ffigen.dart
+    cd ..
+}
+
+function get_dart_deps() {
+    if [ "${_DART_PUB_GET_DONE:-0}" = 0 ]; then
```

Comment:
I guess this will not be needed when we port it to just because the task will be already run only once.

Reply:
yes, this is one of the reasons why I still went with `just` instead of `tusk` even through `just` is a big ... strange.